packages/formik/src/components/FieldAdapter.tsx

```diff
@@ -0,0 +1,62 @@
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+import { Component } from 'react';
+
+export interface Props {
+  name: string;
+  component?: any;
+  onChange?: any;
+  onBlur?: any;
+  value?: string | boolean;
+  checked?: boolean;
+  fieldType?: string;
+  parent?: any;
+  options?: any;
+  attrs?: any;
+}
+
+export default class Field extends Component<Props, {}> {
+  public static contextTypes = {
+    formik: PropTypes.object
+  };
+
+  constructor(public props: Props, public context: any) {
+    super(props, context);
+    if (!context.formik) {
```

Comment:
Do we really need this check here? We don't allow users to create a form manually.

Reply:
It's not necessary. I plan to clean up all components later.